### PR TITLE
Incrementals

### PR DESCRIPTION
**Pull request by v1v**

### CODE REVIEW

**Comment by v1v** [comment]
Any feedback?

**Comment by halkeye** [comment]
@jenkinsci/code-reviewers

**Comment by v1v** [comment]
any feedback?

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,5 @@ target
 
 # virtual machine crash logs, see http://www.java.com/en/download/help/error_hotspot.xml
 hs_err_pid*
+
+.flattened-pom.xml
```
Comment (jglick) [inline .gitignore]: Unnecessary since this is created in the `target` directory.
```suggestion
```

File: pom.xml [hunk 2]
```diff
@@ -5,14 +5,16 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>3.40</version>
+    <version>3.43</version>
```
Comment (jglick) [inline pom.xml]:
```suggestion
    <version>3.46</version>
```

**Comment by v1v** [comment]
@jenkinsci/code-reviewers, any other feedback?  Otherwise, it might be possible to merge this PR?

Thanks

File: pom.xml [hunk 3]
```diff
@@ -5,14 +5,16 @@
...
   </parent>
   <artifactId>display-url-api</artifactId>
-  <version>2.3.2-SNAPSHOT</version>
+  <version>${revision}${changelist}</version>
   <packaging>hpi</packaging>
 
   <properties>
+    <revision>2.3.2</revision>
+    <changelist>-SNAPSHOT</changelist>
     <jenkins.version>2.89.4</jenkins.version>
```
Comment (daniel-beck) [inline pom.xml]: 2.121.x would be better, since that an implied dependency less

Comment (v1v) [reply pom.xml]: Good point, I've just bumped to `2.121.3`

**Comment by v1v** [comment]
I guess I addressed all the comments so far, are we good to merge it?

**Pull request merged**